docs/pages/admin-guides/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,197 @@
+
+For break glass access, the OpenSSH server must be configured to trust client certificates issued by the Teleport Certificate Authority (CA).
+
+1. **Export the Teleport CA public key:**
+
+On the OpenSSH server, run the following command. Replace `proxy.example.com` with the address of your Teleport Proxy Service:
+
+```bash
+export KEY=$(curl 'https://proxy.example.com/webapi/auth/export?type=openssh' | sed "s/cert-authority\ //")
+```
+
+2. Make the public key accessible to sshd:
+
+```bash
+sudo bash -c "echo \"$KEY\" > /etc/ssh/teleport_openssh_ca.pub"
+sudo bash -c "echo 'TrustedUserCAKeys /etc/ssh/teleport_openssh_ca.pub' >> /etc/ssh/sshd_config"
+```
+
+3. Restart the sshd service:
+
+```bash
+sudo systemctl restart sshd
+```
+
+Now, `sshd` will trust users who present a Teleport-issued SSH certificate.
+
+<Admonition type="important">
+Complete Steps 4 and 5 below to ensure only the `breakglass` user is allowed to log in with a valid Teleport certificate and is configured with proper access restrictions, in order to prevent user switching once logged in. 
+</Admonition>
+
+## Step 2/6. Create a `breakglass` user on the Teleport Agent server
+
+Log into the Teleport Agent server and run the following commands.
+
+1. Create the `breakglass` user:
+
+```bash
+sudo adduser breakglass
+```
+
+Set a password for `breakglass` when prompted. We recommend the use of a long, random string as this password will never be used for access.
+
+**Note:** As a general security principle, it is highly recommended to limit `root` permissions through `sudo`. Because `breakglass` will need ability to gain elevated access to the system to troubleshoot and recover access, however, any use of break glass access should always be audited via sshd access logs. 
+
+## Step 3/6. Restrict cert-based SSH access to the `breakglass` user
+
+The `/etc/ssh/authorized_principals` directory is used in conjunction with OpenSSH's certificate-based authentication mechanism to configure fine-grained control over which users or roles are allowed to authenticate using SSH certificates.
+We will use this mechanism to restrict cert-based SSH access only to the `breakglass` user to prevent other Teleport users from being able to generate client certificates and gain access to the server outside of Teleport.
+
+1. Ensure the following line is added and/or uncommented in your `/etc/ssh/sshd_config` file:
+
+```
+AuthorizedPrincipalsFile /etc/ssh/authorized_principals/%u
+```
+
+2. Create an authorized_principals file for the `breakglass` user 
+
+```bash
+echo "breakglass" | sudo tee /etc/ssh/authorized_principals/breakglass
+```
+
+3. Restart the sshd service:
+
+```bash
+sudo systemctl restart sshd
+```
+
+## Step 4/6. Create `breakglass` Role and User in Teleport for impersonation
+
+1. Create `breakglass` Teleport role:
+
+Define the role in a file named breakglass-role.yaml
+
+```yaml
+kind: role
+metadata:
+  name: breakglass
+spec:
+  allow:
+    logins:
+    - breakglass
```

Comment:
If we're going through the trouble of setting up a principals list why are we using a mundane name like `breakglass`? How confident are we that we will never grant it as a regular username to any user of the cluster?